Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/messaging/service/decorator/HttpHeaderServiceEndpointDecorator.java

```diff
@@ -105,7 +105,7 @@ private String getKeepAliveProperty() {
                 addPropertiesToCustomHeaders(allPropNames, customHeaders);
             }
         } catch (PropertyAccessException ex) {
-            LOG.warn("Unable to access properties for custom http headers.", ex.getLocalizedMessage());
+            LOG.warn("Unable to access properties for custom http headers.", ex);
```

Comment:
LOG.warn("Unable to access properties for custom http headers.{}", ex.getLocalizedMessage(),ex);